src/main/frontend/webpack.common.js

```diff
@@ -12,6 +12,9 @@ let config = {
     filename: "js/[name].bundle.js",
     libraryTarget: "amd",
   },
+  optimization: {
+      minimize: false
+  },
```

Comment:
Should be removed (I forgot to delete it)

Reply:
done !